docs/config.json

```diff
@@ -11,10 +11,21 @@
           "slug": "/getting-started/",
            "entries": [
             { "title": "Linux Server", "slug": "/getting-started/linux-server/" },
+            { "title": "Kubernetes", "slug": "/kubernetes-access/getting-started/cluster/" },
             { "title": "Docker Compose", "slug": "/getting-started/docker-compose/" },
             { "title": "DigitalOcean", "slug": "/getting-started/digitalocean/" }
           ]
         },
+        {
+          "title": "Connect your infrastructure",
```

Comment:
Sorry for being unclear in my [first comment](https://github.com/gravitational/teleport/pull/10112#discussion_r799809564). Rather than add the "Connect your infrastructure" and "Getting Started" sections to the nav menu, I think these should be in the "Getting started" landing page (`/docs/getting-started/`). 

When readers visit the page, they would see a section header ("Start here," "Try out Teleport," or something similar) with few guides, so they're not overwhelmed with choice and can get started quickly.

Below that section of the page would be another header, "Connect your infrastructure," with tiles for each of the resources listed here.

The goal would be to communicate in the `/docs/getting-started` page that some guides are for beginners are some are more intermediate, so there's more of a progression in terms of how users go through our guides.

Adding this organization to the nav menu is a bit too cluttered, I think, and I'm not sure readers will notice the progression in the nav menu from the "Getting Started" link to the "Connect your infrastructure" link.

@xinding33, I know you're working on how we organize our Getting Started resources—what are your thoughts here?

Reply:
This is my crappy wireframe.. but what's your thoughts on this?

<img width="953" alt="image" src="https://user-images.githubusercontent.com/559288/155202936-e683dc28-9b31-4edf-8bc5-6753cc456dec.png">

Reply:
Thanks for mocking this up—this is definitely along the lines of what I was thinking. Some more specific thoughts:

- What's the intended difference between the guides in "Start Here" and "Try Teleport"? Is "Start Here" for proofs of concept in staging/production, while "Try Teleport" is for local demo environments? If so, I'm not sure the "Try Teleport Enterprise" link would fit. Also, depending on the organization, I'm guessing that some users may want to try Teleport locally before setting up a more fully realized environment (i.e., they'd start with "Try Teleport").
- In the "Connect your infrastructure" section, I think we can consider removing "Connect...to Teleport" from each tile and leaving only the name of each resource. Otherwise, the labels might get a bit noisy. 

What do you think of the wireframe, @xinding33?